tests/utils/eol-conversion.spec.ts

```diff
@@ -63,7 +63,7 @@ describe('detectEncoding() function', function () {
 	it('should correctly detect the encoding of a few selected files', async () => {
 		const sampleBinary = [
 			'drivelist/build/Release/drivelist.node',
-			'mountutils/build/Release/MountUtils.node',
+			'mountutils/prebuilds/linux-x64/mountutils.node',
```

Comment:
etcher-sdk breaking change (from v9 -> v10) besides deprecating node18 also changes the build of mountutils which changes the final .node binary.

The linux-x64 is prebuilt while the other platforms (windows, marcos etc) are build on the fly on the host machine. As this is a simple test for checking that it parses it is a binary, I use the linux-x64 on all platforms as it is the only one to be certain to be there (as it is prebuilt).

See: https://github.com/balena-io-modules/mountutils/pull/84